spec/lib/traject/psulib_config_spec.rb

```diff
@@ -4,26 +4,37 @@
 
 RSpec.describe 'Psulib_config spec:' do
   let(:leader) { '1234567890' }
+  let(:fixture_path) { './spec/fixtures' }
 
   before(:all) do
     c = './lib/traject/psulib_config.rb'
     @indexer = Traject::Indexer.new
     @indexer.load_config_file(c)
   end
 
-  describe 'Record with music numeric should have semicolons for all but subfield e' do
-    let(:field) { 'music_numerical_ssm' }
-    let(:music_383) do
-      { '383' => { 'ind1' => '1', 'ind2' => '0', 'subfields' => [{ 'b' => 'op. 36' },
-                                                                 { 'b' => 'op. 86' },
-                                                                 { 'b' => 'op. 35' },
-                                                                 { 'e' => 'Bach' },
-                                                                 { 'e' => 'Motzart' }] } }
+  describe 'id' do
```

Comment:
id and music numeric specs just re-ordered right?

Reply:
yeah that's what I meant in the commit message, sorry, not clear.

I also have to say, as a bigger topic, do we want the new spec I added for this? I think the answer is "yes" but we also have dozens of straightforward `extract_marc` fields un-tested.... This is relevant for work required in https://github.com/psu-libraries/psulib_traject/issues/244 too.  Again my leaning is that "yes" they need tests just for clarity. But I dont know that I want to add tests for all of our fields. How much value do we get for these tests?

Reply:
I think we initially decided not to add specs for just straightforward extract_marc fields, and just add ones for that involves additional logic. 
Saying that, I think it is OK even nice to have a spec for a newly added field, kinda gives a piece of mind. 

If we wanna consider about adding a spec for those other fields too, I agree that might be a work that might not hold as much value in the end but maybe we can add several fixtures that captures all the fields and somehow find a way to just test result of the whole import, kinda like a feature specish.

Reply:
think we're on the same page 👍